src/main/java/com/yuichi/assignment6/CatController.java

```diff
@@ -0,0 +1,19 @@
+package com.yuichi.assignment6;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class CatController {
+    @GetMapping("/cat")
+
+    public List<Cat> pet() {
+        List<Cat> cats = List.of(
+                new Cat("tom", "american shorthair"),
+                new Cat("tama", "mike"));
```

Comment:
細かいですが、ここって以下のようにフォーマットされませんかね？👀
気のせいだったら大丈夫です！
```suggestion
                new Cat("tama", "mike")
                );
```